man/systemd-nspawn.xml

```diff
@@ -522,6 +522,21 @@
         <option>--drop-capability=</option>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--network-namespace-path=</option></term>
+
+        <listitem><para>Specify an absolute path to an arbitrary file
```

Comment:
the path doesn't actually have to be absolute, we'll make it absolute as necessary.

Maybe:

> Takes the path to a file representing a kernel networking namespace that the container shall run in. The specified path should refer to a (possibly bind mounted) networking namespace file, as exposed by the kernel below `/proc/$PID/ns/net. …

and so on?